clients/client-cloudfront/test/cloudfront-features.e2e.spec.ts

```diff
@@ -0,0 +1,159 @@
+import { getE2eTestResources } from "@aws-sdk/aws-util-test/src";
+import { CloudFront } from "@aws-sdk/client-cloudfront";
+import { beforeAll, describe, expect, test as it } from "vitest";
+
+describe("@aws-sdk/client-cloudfront", () => {
+  let client: CloudFront;
+  let region: string;
+
+  beforeAll(async () => {
+    const e2eTestResourcesEnv = await getE2eTestResources();
```

Comment:
the e2e test resources should not be invoked for tests that do not need a bucket, identity pool, or MRAP

the region can be assumed to be us-west-2